src/com/rct/homework/lesson2/task07/validator/NoDigitValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.rct.homework.lesson2.task07.validator;
+
+import com.rct.homework.lesson2.task07.localisation.MessageStrings;
+
+/**
+ * Validating strings for containing no numbers.
+ *
+ * @author Oleg Baslak
+ * @version 1.0
+ * @since 08.10.2016
+ */
+public class NoDigitValidator extends Validator {
+
+    //  regex means "no numbers in string"
+    private String symbolsToFind = "\\D*$";
```

Comment:
I think, it matches everything after '3', cause string 
"asdasd" doesn't contain any numbers. 
Input from my computer:
![image](https://cloud.githubusercontent.com/assets/1713494/19224106/b40f24ba-8e87-11e6-9bc3-0c2758e0eed7.png)